src/JsonLd/Context.php

```diff
@@ -138,6 +138,9 @@ private function getContextType($name, array $data = [])
             case 'music_playlist':
                 return new ContextTypes\MusicPlaylist($data);
                 break;
+            case 'corporation':
```

Comment:
@Torann I'd suggest to change this approach and use the class name in the getContextType, like
```
getContextType('Corporation', [])
```
We could check if the class exists and get a new instance instead of using an alias and switch. What do you think?